UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/IWebJobAccessKeyProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public interface IWebJobAccessKeyProvider
+    {
+        public string GetWebJobsAccessKey(string keyName);
+    }
```

Comment:
access modifiers not required in interface for methods

Reply:
Removed access Modifier .